### PR TITLE
Hide milestones in feature list entries for prelaunch features.

### PR DESCRIPTION
**Pull request by jrobbins**
This should resolve issue #984.

In the feature list, a feature entry no longer show milestone values when the feature is in a prelaunch stage.

My understanding is that displaying these milestones is confusing because they are not relevant before launch, even if a developer entered them.

### CODE REVIEW

File: static/elements/chromedash-feature.js
Comment (jpmedley) [inline static/elements/chromedash-feature.js:109]: Shouldn't this be 'Developer trial.'

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: static/elements/chromedash-feature.js
Comment (jrobbins) [reply static/elements/chromedash-feature.js:109]: That change to models.py is still pending in a different CL.  However, thank you for pointing out that this will need to change after that lands.  I'll add a TODO here.

**Comment by jpmedley** [comment]
You should probably also test whether these items show up when navigating via the sidebar. To state another way, if a future version number is hidden, will I still see the feature in the list/search for that version number? 

image

**Review by cwilso — APPROVED** [review]
LGTM, pending answer to Joe's question about if prelaunch features show up in milestone nav when hidden.

**Review by jpmedley — APPROVED** [review]

**Review by cwilso — COMMENTED** [review]
LGTM.

**Comment by cwilso** [comment]
(Sorry, getting review notifications I'm having trouble getting rid of - don't mind me.  :)

**Pull request merged**
